.buildkite/longruns_gpu/pipeline.yml

```diff
@@ -137,6 +137,11 @@ steps:
           CLIMACOMMS_DEVICE: "CUDA"
           LONGER_RUN: ""
 
+      - label: "Send simulation output to Azure VM for visualization"
```

Comment:
The path looks fine! But since this is running when LONGER_RUN = true, we should either make the longer run the p-model (my preference) or else this path should be to snowy_land_longrun_gpu